ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/Util/IxSet.hs

```diff
@@ -4,10 +4,10 @@
 module Ariadne.Wallet.Cardano.Kernel.DB.Util.IxSet (
     -- * Primary keys
     HasPrimKey(..)
-  , OrdByPrimKey -- opaque
+  , OrdByPrimKey (..) -- opaque
```

Comment:
`-- opaque` means that internals (constructor, etc.) are not exported, so this comment is not true anymore.

But actually I don't like that `OrdByPrimKey` stops being opaque. Also I don't like that `IxSet` stops being opaque.
AFAIU, there are two use cases:
1. `IxSet.toList` and apply `unwrapOrdByPrimKey` to all items. I'd add a function to this module which would do it (`toList` and unwrap).
2. `IxSet.toAscList`. I'd also add a similar function. Currently `toAccountsList` returns a list of `OrdByPrimKey`, but I think these `OrdByPrimKey` are not needed outside, it can return simply a list of `HdAccount`.